graphql/directive_test.go

```diff
@@ -53,6 +53,28 @@ func setupPermissions(t *testing.T) {
 	err = roleManager.AddScope(superUserScope)
 	require.NoError(t, err)
 
+	superUserProjectRole := gimlet.Role{
+		ID:    evergreen.SuperUserProjectAccessRole,
+		Name:  "admin access",
+		Scope: evergreen.AllProjectsScope,
+		Permissions: map[string]int{
+			"project_logs":             10,
```

Comment:
nit: as opposed to hardcoding this, preference for using the variable names (evergreen.PermissionLogs, etc)